README.md

```diff
@@ -413,6 +413,7 @@ encryption library providing MD5, SHA1, SHA2 hashing and HMAC functionality, as
 ### Web-tools
 
 - [Boxentriq](https://www.boxentriq.com/code-breaking) - Easy to use tools for analysis and code-breaking of the most frequent ciphers, including Vigenère, Beaufort, Keyed Caesar, Transposition Ciphers, etc.
+- [Crypto Puzzles](http://rupp.de/crypto_puzzles/crypto_puzzles_web.py) - Crypto Puzzles provides a bunch of functions for encryption and stegonagraphy as brain teasers, to get kids interessted in cryptography. ᵺ ⌶ 💲     ℹ️ ى     ɉ ᶙ 💰 ኘ     🍩 ₦ €     モ ᚸ ㏂ 🆊 ᇉ. Code availabe on Github. 
```

Comment:
Let's leave it as `Crypto Puzzles provides a bunch of functions for encryption and stegonagraphy as brain teasers, to get kids interessted in cryptography.`